_vcsh_bash

```diff
@@ -128,11 +132,17 @@ _vcsh () {
 
 	# git command on repository
 	if [[ -n "${repos[$cmd]}" ]]; then
-		_vcsh_git_command $subcword
+		local REPO_DIR="${XDG_CONFIG_HOME:-$HOME/.config}/vcsh/repo.d/${cmd}.git"
+		GIT_DIR=${REPO_DIR} _vcsh_git_command $subcword
 	fi
 	return 0
 }
 
 complete -F _vcsh vcsh
 
 # vim: ft=sh:
```

Comment:
I think VIM modelines typically need to trail emacs ones, they need to be within a certain distance from the end.

Reply:
Ok, I will change the order.